tests/unit/test_types.py

```diff
@@ -0,0 +1,1852 @@
+    None,
+    'AQgAAAACCwEAgANEAAAgAAABAAAASsQF+7cDRAABAA==',
+    None,
+    'AHsUrkfheoQ/AAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAAAAPA/AQAAAAAAAIA=',
+    None,
+    'AAAAAAAAAPA/AAAAAAAA8D8AAAAAAABZQAAAAAAAAPA/AQAAAAAAAAAAAPA/AAAAAAAA8D8=',
+    None
+]
+
+def test_types():
```

Comment:
Please split the tests by type, that will be easier to troubleshoot when things go wrong.

Reply:
Also test using  the flag `experimental_python_types` set to `True` with the `TrinoResult` class.

Reply:
Done